docs/source/upcoming_release_notes/996-ADD_AT2L0_err_support.rst

```diff
@@ -0,0 +1,33 @@
+996 ADD AT2L0 err support
+#################
+
+API Changes
+-----------
+- clear_errors() method to clear errors; e.g. at2l0.clear_errors()
+- print_errors() to print error summary; e.g. at2l0.print_errors()
```

Comment:
Quick clarification: the API change category is for API breaks, e.g. if you removed a method or changed the signature in a backwards-incompatible way. These belong in Device Updates.

Also, the way you've written this here implies that many devices have gotten these methods added, since at2l0 is presented as an example rather than as the only device that has been updated.